hadoop/src/tesser/hadoop.clj

```diff
@@ -200,3 +200,68 @@
                   first)]
         (when (error? x) (throw (ex-info "Hadoop fold error" x)))
         x))))
+
+(defn identity-mapper
+  "Does nothing in the map step."
+  {::mr/source-as :vals
+   ::mr/sink-as   :vals}
+  [records]
+  (r/map identity records))
+
+(defn partition-randomly
+  ^long [key val ^long nparts]
```

Comment:
key is null, the actual content of val is not necessary for assigning randomly. I changed key and val to underscores, is that better? Yes, most examples of partitioning functions I saw declared both the type of nparts and the return type of the partitioning function, to avoid boxing.